app/src/main/java/org/fossasia/openevent/app/common/mvp/view/BaseFragment.java

```diff
@@ -20,10 +20,14 @@ public void onResume() {
         setTitle(getString(getTitle()));
     }
 
-    protected abstract Lazy<P> getPresenterProvider();
+    protected Lazy<P> getPresenterProvider() {
+        return null;
+    }
 
     protected P getPresenter() {
-        return getPresenterProvider().get();
+        if (getPresenterProvider() != null)
+            return getPresenterProvider().get();
```

Comment:
Extract provider to a local var

Reply:
doing so is leading to crash

Reply:
It's impossible. The semantics will remain same

```kotlin
val provider = getPresenterProvider()
return (provider != null) ? provider.get() : null
```